opencog/query/README.md

```diff
@@ -356,23 +353,23 @@ Currently Implemented Algorithm
    Thus, for example: the clause _subj(row, Steve) has _subj at the
    root of the tree, with "row" and "Steve" as leaves.
 
-   Distinct trees may have common vertices.  The common vertices
-   act to join together the trees into a single, connected graph.
-   Disconnected graphs are not allowed; the algorithm will fail to
-   find and ground disconnected components.  The graph as a whole may
-   contain loops (the decomposition into trees keeps the algorithm
-   from having to explicitly accommodate loops).
```

Comment:
Well, you removed the bit about loops. Its important, since an naive implementation results in an infinite loop, if the graph has loops.  The pattern matcher detects and breaks such infinite loops.  There's some unit test for this.